src/main/resources/view/PersonListCard.fxml

```diff
@@ -29,7 +29,6 @@
       </HBox>
       <FlowPane fx:id="tags" />
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
```

Comment:
This is to prevent the GUI from showing even the Address field right?

Reply:
Yes